version.php

```diff
@@ -25,7 +25,7 @@
 defined('MOODLE_INTERNAL') || die();
 
 
-$plugin->version   = 2020121800;
+$plugin->version   = 2022040100;
 $plugin->requires  = 2020061500;
 $plugin->component = 'qtype_oumultiresponse';
 $plugin->maturity  = MATURITY_STABLE;
```

Comment:
Becuase this plugin now uses the get_default_value mechanism that was only added in Moodle 3.11, from this point on, we will only be able to support Moodle 3.11+ with this plugin, so we need to update the release info on the next line to this one. (See also the comment above about increasing the version number to 2.4.)

Also as part of this, we need to update .github/workflows/ci.yml to stop it trying to run the tests with older Moodle versions.